src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -234,6 +247,13 @@ interface IRewardsCoordinator {
         uint256 claimedAmount
     );
 
+    event PerformanceBasedRewardCreated(
+        address indexed avs,
+        uint256 indexed submissionNonce,
+        bytes32 indexed rewardsSubmissionHash,
+        PerformanceRewardsSubmission rewardsSubmission
```

Comment:
Can we emit the RewardType here too? Just because commissions are set based on the RewardType